lib/widgets/layouts/adaptive_layout/app_adaptive_scaffold_body.dart

```diff
@@ -43,6 +43,9 @@ class AppAdaptiveScaffoldBodyController extends State<AppAdaptiveScaffoldBody>
 
   final activeRoomIdNotifier = ValueNotifier<String?>(null);
 
+  final ValueNotifier<bool> autoDisplayPermissionContactDialog =
```

Comment:
-  I don't see this variable update, always is true. 
- Missing dispose 

IMO: No need variable.